matrix-sdk-android/src/androidTest/java/org/matrix/android/sdk/internal/crypto/ExportEncryptionTest.kt

```diff
@@ -83,7 +83,8 @@ class ExportEncryptionTest {
     @Test
     fun checkExportDecrypt1() {
         val password = "password"
-        val input = "-----BEGIN MEGOLM SESSION DATA-----\nAXNhbHRzYWx0c2FsdHNhbHSIiIiIiIiIiIiIiIiIiIiIAAAACmIRUW2OjZ3L2l6j9h0lHlV3M2dx\n" + "cissyYBxjsfsAndErh065A8=\n-----END MEGOLM SESSION DATA-----"
+        val input =
```

Comment:
Why did you add a line break here ?
You didn't add it [L53](https://github.com/vector-im/element-android/blob/3f3662e60562ca76f954c3d7f025a05c5524c584/matrix-sdk-android/src/androidTest/java/org/matrix/android/sdk/internal/crypto/ExportEncryptionTest.kt#L53) and [L68](https://github.com/vector-im/element-android/blob/3f3662e60562ca76f954c3d7f025a05c5524c584/matrix-sdk-android/src/androidTest/java/org/matrix/android/sdk/internal/crypto/ExportEncryptionTest.kt#L68)

Reply:
This is the auto-formatter, presumably because the line went over the char limit.

I manually changed this and the below remark to be multiline (the proper way), but be wary it may still be possible we see this elsewhere in the app